### PR TITLE
Fix nusd Metapool farm card not showing staking Info

### PR DESCRIPTION
**Pull request by memacius32**
Fix nusd Metapool farm card not showing staking Info:

- Replace lpToken generation with token pair by stablepool constant for stablepool farms.

Before:
Screen Shot 2022-06-16 at 18 48 05

After:
Screen Shot 2022-06-20 at 17 01 22

### CODE REVIEW

File: src/state/stake/useFarmContractsForVersion.ts [hunk 1]
```diff
@@ -47,20 +49,26 @@ export function useFarmContractsForVersion(chefVersion: ChefVersions): StakingTr
 
     return lpAddresses.map((lpAddress, index) => {
       // User based info
+
       const userStaked = userInfo[index]
       const activeFarmID = getActiveFarmID(lpAddress)
+
+      const stablePoolFarm = activeFarms[activeFarmID].stableSwapPoolName
+      const stablePool = stablePoolFarm ? STABLESWAP_POOLS[stablePoolFarm] : null
```
Comment (rej156) [inline src/state/stake/useFarmContractsForVersion.ts:57]:
```suggestion
      const stablePool = STABLESWAP_POOLS?.[stablePoolFarm]
```

for educational purposes

**Review by rej156 — APPROVED** [review]
nice one :))

File: src/state/stake/useFarmContractsForVersion.ts (hunk 1 above)
Comment (memacius32) [reply src/state/stake/useFarmContractsForVersion.ts:57]: yeah, but the problem is that:

```
Type 'null' cannot be used as an index type
``` 
there may be a way though

Comment (rej156) [reply src/state/stake/useFarmContractsForVersion.ts:57]:
```suggestion
      const stablePool = stablePoolFarm && STABLESWAP_POOLS?.[stablePoolFarm]
```

Comment (rej156) [reply src/state/stake/useFarmContractsForVersion.ts:57]: lol nah that would actually evaluate as false too kek

Comment (rej156) [reply src/state/stake/useFarmContractsForVersion.ts:57]: rrrreeeeeee

Comment (memacius32) [reply src/state/stake/useFarmContractsForVersion.ts:57]: LOL right, covid getting into my mind sorry, pushing and merging

Comment (memacius32) [reply src/state/stake/useFarmContractsForVersion.ts:57]: > lol nah that would actually evaluate as false too kek

works! :)

**Pull request merged**